static/js/ui-utils.js

```diff
@@ -3,119 +3,7 @@
 var openDraw = null;
 
 function rawBGsEnabled() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['rawBGsEnabled' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/799258312/issues/source?bid=2452379&fileBranchId=2725007#l5)